TileFormats/BatchTable/README.md

```diff
@@ -150,6 +150,423 @@ var cartographicArray = new Float64Array(batchTableBinary.buffer, byteOffset, ca
+        }
+      },
+      {
+        "name" : "Car",
+        "length" : 3,
+        "instances" : {
+          "carType" : ["truck", "bus", "sedan"],
+          "carColor" : ["red", "blue", "white"]
+        }
+      },
+      {
+        "name" : "Tree",
+        "length" : 2,
+        "instances" : {
+          "treeHeight" : [10, 15],
+          "treeAge" : [5, 8]
+        }
+      }
+    ],
+    "instancesLength" : 8,
+    "classIds" : [0, 0, 0, 1, 1, 1, 2, 2]
+  }
+}
+```
+
+`classes` is an array of objects, where each object contains the following information:
+* `name` - the name of the class
+* `length` - the number of instances of the class
+* `instances` - metadata for the instances. This section is similar to a standard batch table; properties may be stored as an array of values or a reference to data in the binary body.
+
+`instancesLength` is the number of instances. In this example `instancesLength` equals `batchLength`, however this is not always the case (as will be seen below).
+
+`classIds` is an array of integers of length `instancesLength`. Each `classId` specifies the instances's class as an index in the `classes` array.
+
+In the example above 0 indicates a "Lamp" instance, 1 indicates a "Car" instance, and 2 indicates a "Tree" instance.
+
+A feature's `batchId` is used to access its class in the `classIds` array. Therefore features with a `batchId` of 0, 1, 2 are "Lamp" instances, features with a `batchId` of 3, 4, 5 are "Car" instances, and features with `batchId` of 6, 7 are "Tree" instances.
+
+Note that the batch table hierarchy does not directly provide an instances's index into its class's `instances` array. Instead the index can be inferred by the number of instances with the same `classId` that have appeared before it. An implementation may want to compute these indices at load time so that property access is as fast as possible.
+
+To put this more concretely using the current example:
+
+The feature with `batchId = 0` is the first "Lamp" instance, and its properties are:
+
+```
+lampStrength : 10
+lampColor : "yellow"
+```
+
+The feature with `batchId = 2` is the third "Lamp" instance, and its properties are:
+
+```
+lampStrength : 7
+lampColor : "white"
+```
+
+The properties for `batchId = 7` is the second "Tree" instance, and its properties are
+
+```
+treeHeight : 15
+treeAge : 8
+```
+
+Finally, `classIds` may be a reference to data in the binary body. If omitted, `componentType` defaults to `UNSIGNED_SHORT`. `type` is implicitly `SCALAR`.
+
+```json
+"classIds" : {
+    "byteOffset" : 0,
+    "componentType" : "UNSIGNED_SHORT"
+};
+```
+
+### Instance Hierarchy
+
+Another limitation of the standard batch table is the difficulty in expressing metadata hierarchies.
+
+For example consider a tile that represents a city block. The block itself contains metadata, the individual buildings contain metadata, and the building components (doors, walls, roofs) contain metadata.
```

Comment:
Perhaps a screenshot or a series of screenshots with different features highlighted from the Sandcastle example would be useful here.